<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gusuarios</class>
 <widget class="QDialog" name="Gusuarios">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>797</width>
    <height>391</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestionar Usuarios</string>
  </property>
  <widget class="QGroupBox" name="groupBox8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>751</width>
     <height>351</height>
    </rect>
   </property>
   <property name="title">
    <string>Gestionar Usuarios</string>
   </property>
   <widget class="QTableView" name="lista8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>721</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>280</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="addbutton8">
       <property name="text">
        <string>Adicionar</string>
       </property>
       <property name="icon">
        <iconset resource="pandasoft.qrc">
         <normaloff>:/images/list-add.png</normaloff>:/images/list-add.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="modbutton8">
       <property name="text">
        <string>Modificar</string>
       </property>
       <property name="icon">
        <iconset resource="pandasoft.qrc">
         <normaloff>:/images/document-edit.png</normaloff>:/images/document-edit.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="delbutton8">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Eliminar</string>
       </property>
       <property name="icon">
        <iconset resource="pandasoft.qrc">
         <normaloff>:/images/edit-delete.png</normaloff>:/images/edit-delete.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>320</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="volverboton">
       <property name="toolTip">
        <string>Volver a la Pantalla Principal</string>
       </property>
       <property name="statusTip">
        <string>Volver a la Pantalla Principal</string>
       </property>
       <property name="text">
        <string>Volver</string>
       </property>
       <property name="icon">
        <iconset resource="pandasoft.qrc">
         <normaloff>:/images/go-previous.png</normaloff>:/images/go-previous.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="informacion8">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>310</y>
     <width>112</width>
     <height>136</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>44</height>
    </size>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetClosable</set>
   </property>
   <property name="windowTitle">
    <string>Informacion</string>
   </property>
   <widget class="QWidget" name="informacion8_2">
    <widget class="QLabel" name="informacionlabel8">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="pandasoft.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>volverboton</sender>
   <signal>clicked()</signal>
   <receiver>Gusuarios</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>416</x>
     <y>352</y>
    </hint>
    <hint type="destinationlabel">
     <x>128</x>
     <y>384</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
